### PR TITLE
Adjustments to installing dependencies

### PR DESCRIPTION
**Pull request by bkuhl**
Small changes to composer dependencies in CI

### CODE REVIEW

File: infrastructure/fpm/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,4 @@
 FROM realpage/php:7.0-fpm
-MAINTAINER RealPage DevOps foundation-devops@indatus.com
```
Comment (jlaswell) [inline infrastructure/fpm/Dockerfile]: Why remove this?

Comment (bkuhl) [inline infrastructure/fpm/Dockerfile]: The built container will never be anything that devops maintains... it's only a template.  This should really be the team's email address who's maintaining the product.

Comment (jlaswell) [inline infrastructure/fpm/Dockerfile]: Ah, good call!

**Comment by jlaswell** [comment]
:+1:

**Pull request merged**